app/state_machines/configurable_state_machine/machine.rb

```diff
@@ -101,7 +101,7 @@ def key_present_but_nil?(config, key)
     # * :acting_user (this corresponds to the current_user)
     # * :acting_team
     #
-    #rubocop:disable Metrics/CyclomaticComplexity
+    #rubocop:disable Metrics/CyclomaticComplexity, Metrics/MethodLength
```

Comment:
This method should be re-factored to extract code common to can_trigger_event?() and possibly event_present_and_triggerable?().